requirements.txt

```
@@ -16,7 +16,7 @@ grpcio-tools==1.68.0
     # via kuksa-client
 jsonpath-ng==1.7.0
     # via kuksa-client
-kuksa-client==0.4.3
+kuksa-client==0.5.1-a2
```

Change this once KUKSA Python SDK is released
```suggestion
kuksa-client==0.5.1
```